src/main/scala/io/citrine/lolo/TrainingResult.scala

```diff
@@ -39,5 +39,5 @@ trait MultiTaskTrainingResult extends TrainingResult {
 
   def getModels(): Seq[Model[PredictionResult[Any]]]
 
-  override def getPredictedVsActual(): Option[Seq[(Vector[Any], Seq[Any], Seq[Any])]] = None
+  override def getPredictedVsActual(): Option[Seq[(Vector[Any], Seq[Option[Any]], Seq[Option[Any]])]] = None
```

Comment:
Because not every training datum needs to have a value for every label, this interface needs to expose `Seq[Option[Any]]`